Gemfile.lock

```diff
@@ -405,69 +409,68 @@ GEM
       terminal-table (>= 1.4.0)
       thor (>= 0.16.0)
     jmespath (1.6.2)
-    json (2.6.3)
-    jsonpath (1.1.2)
+    json (2.7.1)
+    jsonpath (1.1.5)
       multi_json
     jwt (2.7.1)
     kubeclient (4.11.0)
       http (>= 3.0, < 6.0)
       jsonpath (~> 1.0)
       recursive-open-struct (~> 1.1, >= 1.1.1)
       rest-client (~> 2.0)
-    large_object_store (1.3.4)
-    llhttp-ffi (0.4.0)
+    large_object_store (1.7.0)
+      zstd-ruby (~> 1.5.5)
+    llhttp-ffi (0.5.0)
       ffi-compiler (~> 1.0)
       rake (~> 13.0)
-    lograge (0.12.0)
+    lograge (0.14.0)
       actionpack (>= 4)
       activesupport (>= 4)
       railties (>= 4)
       request_store (~> 1.0)
     logstash-event (1.2.02)
-    loofah (2.21.4)
+    loofah (2.22.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.12.0)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
     marcel (1.0.2)
     marco-polo (2.0.1)
       railties (>= 6)
-    maxitest (3.3.0)
-      minitest (>= 5.0.0, < 5.12.0)
-    metaclass (0.0.4)
-    method_source (0.8.2)
-    mime-types (3.4.1)
+    maxitest (3.7.0)
+      minitest (>= 5.0.0, < 5.15.0)
+    method_source (1.0.0)
+    mime-types (3.5.2)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2023.0218.1)
+    mime-types-data (3.2024.0206)
     mini_mime (1.1.5)
     mini_portile2 (2.8.5)
-    minitest (5.11.3)
-    minitest-rails (6.1.0)
+    minitest (5.14.4)
+    minitest-rails (6.1.1)
       minitest (~> 5.10)
       railties (~> 6.1.0)
     mixlib-shellout (3.2.7)
       chef-utils
-    mocha (1.2.1)
-      metaclass (~> 0.0.1)
+    mocha (1.16.1)
```

Comment:
### [mocha](https://mocha.jamesmead.org)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.2.1 → 1.16.1

[_~~change-log~~, [source-code](http://github.com/freerange/mocha), [gem-diff](https://my.diffend.io/gems/mocha/1.2.1/1.16.1)_]





<details>
<summary>Commits</summary>

A change of **656** commits. See the full changes on [the compare page](https://github.com/freerange/mocha/compare/v1.2.1...v1.16.1).

These are the first 10 commits:
- (54f38b0) [Simplify Cardinality#invoked_times](https://github.com/freerange/mocha/commit/54f38b079e726ff44f9d4dcde9ebe7c81a1f2c13)
- (290f390) [Use consistent phrasing for configured & invoked times](https://github.com/freerange/mocha/commit/290f390ebdd5398c6188671d213317d96fe2316a)
- (ad0fa3c) [Reduce test's knowledge of irrelevant details](https://github.com/freerange/mocha/commit/ad0fa3c12769a2c16bf6d4619308a1619c85524d)
- (9e449b0) [Record result of invocation before 'returning' it](https://github.com/freerange/mocha/commit/9e449b0c79d41d7045c49a1af648dfe89edcf31e)
- (4470825) [Encapsulate invocations inside cardinality](https://github.com/freerange/mocha/commit/44708253962238e7ecfb14701c908f7a4a8ad6be)
- (feb9c07) [Rename block argument in Invocation#call](https://github.com/freerange/mocha/commit/feb9c070f18406878df2b4dd2ce87070603ec189)
- (a8f7d0a) [Switch args for Invocation constructor & #call](https://github.com/freerange/mocha/commit/a8f7d0ac1dc4fb1a7f5b4dcdbb2a9bce7bbaf5da)
- (ffb16e9) [Only need to use ParametersMatcher in Invocation#mocha_inspe…](https://github.com/freerange/mocha/commit/ffb16e922272b23dcf2a940a3703b3bb63848bc5)
- (4c24a3c) [Pass Invocation to Expecation#match?](https://github.com/freerange/mocha/commit/4c24a3c1f3f0a5b5c4418865c6e6b0fe94611687)
- (3f1b769) [Temporarily make Invocation#arguments externally immutable](https://github.com/freerange/mocha/commit/3f1b7699d6bb4a5c0641d2aadbd7fbeab21ec568)


</details>